scripts/commitcheck.sh

```diff
@@ -16,7 +16,7 @@ function test_url()
 # check for a tagged line
 function check_tagged_line()
 {
-    regex='^\s*'"$1"':\s+\S+\s+\<\S+\>'
+    regex='^\s*'"$1"':\s[-A-Za-z0-9 ]+\s<\S+>$'
```

Comment:
Currently, the email portion of the regex might be too broad. It currently doesn't enforce that `@` is present, it just allows 1 or more non-whitespace characters to exist between the `< >`. Do you want me to enforce that what shows up there is some form of email?

I'm also broadening the name portion of the regex.